lib/lita/adapters/slack/api.rb

```diff
@@ -70,15 +74,22 @@ def set_topic(channel, topic)
         end
 
         def rtm_start
-          response_data = call_api("rtm.start")
+          rtm_response_data = call_api("rtm.connect")
```

Comment:
This is because there are multiple references to "response data" now and the extra prefix differentiates it from user_response_data